app/lib/CaptureRecorder.js

```diff
@@ -61,9 +61,13 @@ CaptureRecorder.prototype.record = function record(req, res) {
     const filePath = resolveFilePath(this.capturePath, url);
     response.url = url;
     response.latency = now() - startTime;
-    writeFile(filePath, response);
-    this.app.log(['record', 'response', 'saved'], url);
-    ++this.count;
+    writeFile(filePath, response, () => {
+      if (error) return this.app.log(['record', 'response', 'error'], error);
+
+      this.app.log(['record', 'response', 'saved'], url);
+
+      ++this.count;
+    });
   }).pipe(res);
```

Comment:
If may be possible to pipe the response stream to both file and mockyeah response, removing the need for `.writeFile()`.